From 91c0544e374a786a179f39f0b3bad166dfc96771 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 9 Feb 2022 14:04:15 +0100
Subject: [PATCH] suricata-bin: add -fcommon to TARGET_CFLAGS

Suppress multiple warnings during compilation:
multiple definition of `default_packet_size'; alert-debuglog.o:(.bss+0x13d44): first defined here
---
 net/suricata-bin/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/net/suricata-bin/Makefile b/net/suricata-bin/Makefile
index d9ca5baef..db46a0dda 100644
--- a/net/suricata-bin/Makefile
+++ b/net/suricata-bin/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=suricata-bin
 PKG_VERSION:=4.0.7
-PKG_RELEASE=5
+PKG_RELEASE:=6
 
 PKG_SOURCE:=suricata-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.openinfosecfoundation.org/download/
@@ -28,6 +28,9 @@ PKG_BUILD_PARALLEL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
+TARGET_CFLAGS += \
+        -fcommon
+
 CONFIGURE_ARGS+= \
 	--enable-nflog \
 	--enable-nfqueue \
-- 
2.32.0

